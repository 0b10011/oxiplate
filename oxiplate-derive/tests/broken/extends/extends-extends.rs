use oxiplate_derive::{Oxiplate};

#[derive(Oxiplate)]
#[oxiplate_inline(r#"{% extends "extends-wrapper.html.oxip" %}{% extends "extends-wrapper.html.oxip" %}"#)]
struct Data {
    title: &'static str,
}

fn main() {
    let data = Data {
        title: "Double extends",
    };

    print!("{}", data);
}
