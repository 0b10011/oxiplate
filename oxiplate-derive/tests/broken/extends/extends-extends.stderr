error: Only block statements are allowed here, along with comments and whitespace.
 --> tests/broken/extends/extends-extends.rs:4:66
  |
4 | #[oxiplate_inline(r#"{% extends "extends-wrapper.html.oxip" %}{% extends "extends-wrapper.html.oxip" %}"#)]
  |                                                                  ^^^^^^^
